mongowire/wireutil.go

```diff
@@ -54,3 +57,13 @@ func writeCString(s string, buf []byte, loc *int) {
 	buf[*loc] = 0
 	*loc = *loc + 1
 }
+
+func getDocSize(doc *bsonx.Document) int {
+	size, _ := doc.Validate()
```

Comment:
Is it okay to be ignoring these errors in `getDocSize()` and `writeDocAt()`? Does `bsonx.ReadDocument()` already validate the document?